src/test/scopes/js/import-matchers/js-named-import-matcher.test.ts

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright IBM Corp. 2023, 2024
```

Comment:
For these net-new test files, make sure the date at the top is `2024, 2024`. Unless they're refactors of the jsx stuff. It's hard to tell from here since git isn't treating it as a rename if it is one for a lot of these files.